assets/js/modules/pagespeed-insights/datastore/report.js

```diff
@@ -89,9 +91,81 @@ const baseSelectors = {
 
 		return reports[ `${ strategy }::${ url }` ];
 	},
+
+	/**
+	 * Gets report audits for the given strategy and URL.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(Object|undefined)} Report audits.
+	 */
+	getAudits: createRegistrySelector( ( select ) => ( state, url, strategy, withStackPacks = false ) => {
```

Comment:
I'd recommend to not introduce a boolean parameter here - for such modifications it's a more clear API to have a separate selector. Furthermore, what we need here in our specific case is the audits that have a _WordPress_ stack pack. Other stack packs might be included in the response data, but we only care about the WordPress one.

So I suggest the following:
* Remove the `withStackPacks` parameter.
* Introduce a separate registry selector `getAuditsWithStackPack( url, strategy, stackPackID )` which calls `getAudits( url, strategy )` and then filters the results by only the audits that have a stack pack description from the `stackPackID` stack pack (in our case we'd pass `wordpress`).

Reply:
Yep, I like it. Added the new `getAuditsWithStackPack` selector.